Services/Mail/PRIVACY.md

```diff
@@ -0,0 +1,80 @@
+# Mail Service Privacy
+
+This documentation does not warrant completeness or correctness. Please report any
+missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de)
+or contribute a fix via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+## General Information
+
+- The `Mail Service` can be disabled in the global administration or restricted to user accounts assigned
+  to certain roles.
+- The delivery to external email addresses via `SMTP` or `Sendmail` can be enabled and configured in the
+  global administration. External email delivery (concrete: using explicit email addresses as recipient) can be
+  also restricted to certain roles.
+- Consumers might use the `Mail Service` to send emails to external email addresses (directly) or use the
+  channelled message delivery of the internal API without considering any activation.
+  Please check the documentation of the respective consumer.
```

Comment:
Please list respective consumers thus we can late put links here.

Reply:
Hmmm, there is no elaborated registry of these consumers. Each component or plugin can use the `Mail Service`.

Of course I could `grep` for all components using the keyword `mail` and provide a static list here. But this list will be hard to maintain IMO. Please advice.

Reply:
I agree that a list of consumers is time-consuming to keep up to date.
Perhaps it would be better to include a corresponding reference to here in each Privacy.md of the potential comsumers.